crates/apollo-router-core/Cargo.toml

```diff
@@ -25,6 +25,7 @@ once_cell = "1.8.0"
 router-bridge = { git = "https://github.com/apollographql/federation.git", rev = "3aa8f3a533f19e31ab984c87a0674ec78c42ebb6" }
 serde = { version = "1.0.130", features = ["derive", "rc"] }
 serde_json = { version = "1.0.71", features = ["preserve_order"] }
+test-log = { version = "0.2.8", default-features = false, features = ["trace"] }
```

Comment:
Maybe it should be in dev-dependencies?

Reply:
oh indeed, lemme make sure it's not transitively used in apollo-router tests and apply this!

Reply:
Also there's test-env-log to remove then on line 40

Reply:
it passed!